src/Components/App.js

```diff
@@ -6,20 +6,19 @@ import Overlay from './Overlay';
 import TileContainer from './data/TileContainer';
```

Comment:
I forgot to move TileContainer outside of our data directory.  We should probably put it in our components directory instead.